triagebot.toml

```diff
@@ -1185,6 +1185,12 @@ project-exploit-mitigations = [
 "/src/rustdoc-json-types" =                              ["rustdoc"]
 "/src/stage0" =                                          ["bootstrap"]
 "/tests/run-make" =                                      ["@jieyouxu"]
+"/tests/rustdoc" =                                       ["rustdoc"]
+"/tests/rustdoc-gui" =                                   ["rustdoc"]
```

Comment:
Is `rustdoc-ui` group working in this file? Because if so I think it would make more sense here and for the search tests (js and js-std).

Reply:
`rustdoc-frontend`?

Reply:
Yes this one. Got confused with the label. ^^'

Reply:
> Is rustdoc-ui group working in this file?

Nope, we only have one for all of rustdoc.

https://github.com/rust-lang/rust/blob/2128d8df0e858edcbe6a0861bac948b88b7fabc3/triagebot.toml#L1028-L1032

Reply:
Sadness. :'(